docs/tutorial.md

```diff
@@ -1,3 +1,117 @@
-# Tutorial
+# Tutorial: using the AWS plugins for Backstage
 
-Coming soon!
+This tutorial walks you through an example of using the AWS plugins for Backstage.
+
+Tutorial steps:
+
+<!-- toc -->
+
+1. [Set up your Backstage app](#set-up-your-backstage-app)
+1. [Clone this repository](#clone-this-repository)
+1. [Create prerequisite AWS resources](#create-prerequisite-aws-resources)
+1. [Run the Backstage app locally](#run-the-backstage-app-locally)
+1. [Explore the Plugins](#explore-the-plugins)
+1. [Cleanup](#cleanup)
+<!-- tocstop -->
+
+## Set up your Backstage app
```

Comment:
We can remove this since the repo comes with a full Backstage app, unlike the old proton repo

Reply:
How about we replace this with pre-requisites for the user, which would be:

1. AWS account
2. Node.js installed (>= 20)
3. CDK CLI installed